tests/behat/features/bootstrap/PostContext.php

```diff
@@ -74,6 +74,26 @@ public function createGroupsWithOwner(TableNode $postsTable) {
       $this->posts[$post->id()] = $post;
     }
   }
+  /**
+   * Create multiple posts at the start of a test with defined users as
+   * authors.
+   *
+   * Creates post of a given type provided in the form:
+   * | field_post     |  author | type | field_visibility | status | langcode
+   * | My description |  Mark   | post | 0                | 1      | en
+   * | ...            | ...     | ...              | ...    | ...
+   *
+   * @Given posts with defined author:
+   */
+  public function createGroupsWithDefinedOwner(TableNode $postsTable) {
```

Comment:
@volodymyr-sydor I see that method is above named as Groups, but it looks like it was done by mistake. So I would propose to rename this one to `createPosts` and also one above to the `createPostsWithOwner`

Reply:
@ribel 
We already have such method - createGroupsWithOwner but for our case, we need to have a defined owner and i would leave a similar name - createGroupsWithDefinedOwner